.clang-tidy

```diff
@@ -24,7 +24,8 @@ Checks: >
     -readability-function-cognitive-complexity,
     -modernize-macro-to-enum,
     -llvm-else-after-return,
-    -readability-else-after-return
+    -readability-else-after-return,
+    -misc-include-cleaner
```

Comment:
Just FYI (feel free to resolve anytime):
We pushed a new build-base image yesterday and it seems a new clang-tidy version with this new rule was installed there. All detected "errors" are unrelated to this PR, so its fine to disable it for now.

Reply:
Yes, I figured that this is the case.
Anyway, I don't think we should enforce it.

Reply:
Agreed.